stdlib/3/asyncio/events.pyi

```diff
@@ -192,18 +192,13 @@ class AbstractEventLoop(metaclass=ABCMeta):
                                 family: int = ..., flags: int = ..., sock: socket = ..., backlog: int = ...,
                                 ssl: _SSLContext = ..., reuse_address: Optional[bool] = ..., reuse_port: Optional[bool] = ...,
                                 ssl_handshake_timeout: Optional[float] = ..., start_serving: bool = ...) -> AbstractServer: ...
-        @abstractmethod
-        async def create_unix_connection(self, protocol_factory: _ProtocolFactory, path: str, *, ssl: _SSLContext = ...,
-                                         sock: Optional[socket] = ..., server_hostname: str = ...,
+        async def create_unix_connection(self, protocol_factory: _ProtocolFactory, path: Optional[str] = ..., *, ssl: _SSLContext = ...,
```

Comment:
Where do you get `path` being Optional from? The abstract classes don't have types, and I looked at the unix event loop for a bit and it also didn't look like it can take None.

Reply:
It's the default value of `path` in both asyncio.events and asyncio.unix_events. I didn't read the code too carefully, but looks like `path` can be None so long as `sock` is not also None (this is the potential false-negative-reducing overloading I alluded to).
https://github.com/python/cpython/blob/master/Lib/asyncio/unix_events.py#L219

Reply:
I see, I should stop looking at 3.6 where there was no default :)

https://github.com/python/cpython/blob/3.6/Lib/asyncio/unix_events.py#L225

Reply:
Oh yeah, I got a little lazy there: https://github.com/python/typeshed/pull/3931/files#diff-b6942d44740a3da4e43f702e0fa4c78f